python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/traffic-policies.j2

```diff
@@ -55,8 +51,8 @@
 {%         for policy in traffic_policies.policies | arista.avd.natural_sort('name') %}
 
 ##### {{ policy.name }}
-
 {%             if policy.matches is arista.avd.defined %}
+
```

Comment:
the logic below to match protocols and source/destination port / field is wrong.

for the source/destionation port/field the key is constantly overwritten in the for loop so it fails if there are multiple protocols. Need to rewrite the logic

Reply:
updated